### DISCUSSION TITLE
subscribe to a particular issue regardless of repo subscription status

### DISCUSSION THREAD

**Discussion by surajsharma** [community/community#51581] [New to GitHub]
hi,

i was wondering if it is possible to subscribe to an invdividual issue on a GH repo with/out subscribing to them all

**Comment by queenofcorgis** [comment]
👋🏼 Hello!

There should be a button within the Issue itself (on the right side of the screen) that says subscribe - that will only subscribe you to notifications on the Issue thread.

**Comment by SMelidoni** [comment]
Hi @surajsharma 👋🏽

Yes, it is possible to subscribe to an individual issue on a GitHub repository without subscribing to all issues.

To subscribe to an individual issue, you can click the "Subscribe" button at the top of the issue page, next to the "Unsubscribe" button. This will subscribe you to that specific issue, and you will receive notifications for updates on that issue.

You can also manage your notification settings for individual issues or all issues on a repository by going to the "Notifications" tab on your GitHub account settings. From there, you can choose which notifications you want to receive and how frequently you want to receive them.
